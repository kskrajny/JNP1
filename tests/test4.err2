Error in line 13: ? a_^ 1 b 2 c
