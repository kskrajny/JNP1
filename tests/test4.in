biletD 5.00 50
biletA 4.00 10
bilet    SPACJEEEEE      0.50 1

1 10:00 a_^ 10:10 b 10:20 c
2 10:10 b 10:20 d 10:30 e
3 10:20 c 10:50 f
4 6:00 a_^ 6:01 b

? a_^ 004 b
? a_^ 1 b
? a_^ 1 b 2 e
? a_^ 1 b 2 c
? a_^ 1 c
? a_^ 1 c 3 f
? b 01 b
