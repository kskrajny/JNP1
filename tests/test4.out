! bilet    SPACJEEEEE     ; bilet    SPACJEEEEE     
! bilet    SPACJEEEEE     ; biletA
! biletD
! biletD
! bilet    SPACJEEEEE     ; biletD
! bilet    SPACJEEEEE     
9